modules/unsupported/process-feature/src/main/java/org/geotools/process/vector/PointStackerProcess.java

```diff
@@ -143,11 +144,16 @@ public SimpleFeatureCollection execute(
             normalize = argNormalize;
         }
 
+        boolean weightClusterPosition=false;
+        if (argWeightClusterPosition!=null){
```

Comment:
Formatting rules not respected. Can you format the lines you added/changed (and only those) based on the official formatting guidelines? See http://docs.geotools.org/latest/developer/conventions/code/style.html